tests/dev/script-mean-model-workflow-existing-results.R

```diff
@@ -0,0 +1,19 @@
+rm(list = ls())

```

Comment:
@pchelle What does this line do? Just curious

Reply:
This line clears the current working environment.
I use it to make sure that previous tests won't interfere with the current script.
Now that you point this out, it is definitely not necessary for the package and should be removed